### DISCUSSION TITLE
[vue] Feature parity between Vue and React

### DISCUSSION THREAD

**Discussion by arturcarvalho** [floating-ui/floating-ui#2271] [Q&A] [answered]
Hi there,

Is there feature parity between `@floating-ui/react` and `@floating-ui/vue`?

Thanks in advance and thank you so much for the library!
Artur

**Comment by atomiks** [comment, marked as answer]
There is feature parity between `vue` and `react-dom`, but not `react`. Vue doesn't have the interactions

**Comment by arturcarvalho** [comment]
Thank you very much for the answer.
